federation/backends/collections.md

```diff
@@ -11,13 +11,25 @@ The 'experimental' flag from https://stac-extensions.github.io/version/v1.1.0/sc
 A collection can be experimental either because there are issues with the actual data or catalog, or because of backend specific issues that make the use of the collection unstable. 
 A collection marked as experimental does not need to comply with further requirements. 
 
-Collections need to indicate the key 'providers' that are responsible for ensuring access to the data and continuity in the case of active missions. The user may depend on the guarantees offered by these
+### Requirements for non-experimental collections
+
+The main goal for non-experimental features is to achieve high user satisfaction by providing stability & usability _and_ to comply with federation agreements that make it possible to offer a unified service.
+The list below makes this more concrete, but is not expected to cover all aspects. It is therefore important that providers, who know their implementations and datasets best, properly judge if a specific collection can be
+considered non-experimental.
+
+1. Collections need to indicate the key 'providers' that are responsible for ensuring access to the data and continuity in the case of active missions. The user may depend on the guarantees offered by these
 providers with respect to the properties (timeliness, completeness,...) of a specific collection. 
 The providers with role 'host' and 'producer' are mandatory.
 
-The collection description needs to specify known limitations with respect to the original collection. For instance, if only a subset of the full archive is available, this should be indicated.
+2. The collection description needs to specify known limitations with respect to the original collection. For instance, if only a subset of the full archive is available, this should be indicated.
```

Comment:
```suggestion
2. The collection description and extents need to specify known limitations with respect to the original collection. For instance, if only a subset of the full archive is available, this should be indicated.
```

Reply:
@m-mohr  are you suggesting that spatial extents need to be a very complex geometry indicating exactly where data may be found at some point in time? Because that's a difficult thing, and would blow up collection metadata.

Reply:
No, more high-level. If you have Australia and Europe covered, specify two bounding boxes, not "the whole world" (or the union of both). I just mean reasonable extents when you have clusters of data, let's say up to 10 or whatever number that seems reasonable.

Reply:
updated